City-of-Helsinki/hitas
# Frontend misc fixes

# Hitas Pull Request

# Description

- Lots of minor enhancements
- Style refactoring

## Pull request checklist

Check the boxes for each DoD item that has been completed:

- **Testing**
- **Database**
- **Documentation**

## Test plan

- Check changed things from commits, see that they are not broken
- See that styles are about the same as before

## Tickets

This pull request resolves all or part of the following ticket(s): -
